ci-operator/jobs/infra-periodics.yaml

```diff
@@ -308,6 +308,71 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  labels:
+    ci.openshift.io/role: infra
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  cron: "30 10 * * 1"  # Run at 10:30 UTC each Monday
```

Comment:
Wow, we're _really_ automating @petr-muller

Reply:
:stuck_out_tongue: